Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -109,6 +109,7 @@ CSSParserContext::CSSParserContext(const Document& document, const URL& sheetBas
     , grammarAndSpellingPseudoElementsEnabled { document.settings().grammarAndSpellingPseudoElementsEnabled() }
     , customStateSetEnabled { document.settings().customStateSetEnabled() }
     , thumbAndTrackPseudoElementsEnabled { document.settings().thumbAndTrackPseudoElementsEnabled() }
+    , lightDarkEnabled { document.settings().cssLightDarkEnabled() }
```

Comment:
I would enable this in UA sheets, I could see this being useful there.

https://searchfox.org/wubkat/source/Source/WebCore/css/parser/CSSParserContext.cpp#56-69

Reply:
Enabled there.